scripts/us_bls/jolts/import.config

```diff
@@ -0,0 +1,18 @@
+import: {
+  import_name: "BLS_JOLTS"
+  provenance_url: "https://bls.gov"
+  provenance_description: "U.S. Bureau of Labor Statistics"
```

Comment:
+ "data on job openings, hires, and separations."

Reply:
Added.